src/main/java/hudson/plugins/parameterizedtrigger/BuildTriggerConfig.java

```diff
@@ -109,24 +115,38 @@ public boolean getTriggerWithNoParameters() {
 
     /**
      * @deprecated
-     *      Use {@link #getProjectList(ItemGroup, EnvVars)}
+     *      Use {@link #getProjectListGeneral(ItemGroup, EnvVars)}
      */
     public List<AbstractProject> getProjectList(EnvVars env) {
-        return getProjectList(null,env);
+        return getProjectList(null, env);
     }
 
     /**
+     * Deprecated: get all projects that are AbstractProject instances
      * @param env Environment variables from which to expand project names; Might be {@code null}.
      * @param context
      *      The container with which to resolve relative project names.
+     * @deprecated
+     *      Use {@link #getProjectListGeneral(ItemGroup, EnvVars)}
      */
+    @Deprecated // Prefer getProjectListGeneral since it can return implementations of the more general Job class
 	public List<AbstractProject> getProjectList(ItemGroup context, EnvVars env) {
-        List<AbstractProject> projectList = new ArrayList<AbstractProject>();
-        projectList.addAll(Items.fromNameList(context, getProjects(env), AbstractProject.class));
-		return projectList;
+        return Util.filter(getProjectListGeneral(context,env), AbstractProject.class);
 	}
 
     /**
+     * Get list of all projects, including workflow job types
+     * @param env Environment variables from which to expand project names; Might be {@code null}.
+     * @param context
+     *      The container with which to resolve relative project names.
+     */
+    public List<Job> getProjectListGeneral(ItemGroup context, EnvVars env) {
+        List<Job> projectList = new ArrayList<Job>();
+        projectList.addAll(Items.fromNameList(context, getProjects(env), Job.class));
```

Comment:
Are you sure?  I ask because the Items.fromNameList logic was present before my commits (I did not change that) and as far as I can tell, dates back to at least 2012.